util.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
```

Comment:
https://harness.drone.io/drone-plugins/drone-slack/57/1/2 looks like there is a minor linting issue with this and go 1.19, should be a quick change.
or ping me and i can change it